fractions/opentracing/src/main/java/org/wildfly/swarm/opentracing/runtime/OpenTracingInstaller.java

```diff
@@ -0,0 +1,37 @@
+package org.wildfly.swarm.opentracing.runtime;
```

Comment:
It might be needed after all. I tried removing this `Installer` class, but the context listener is never called. I also tried removing the `@Vetoed` from the listener, as well as adding a `beans.xml` to `META-INF`. Perhaps I'm missing something ? 

https://gist.github.com/jpkrohling/789eadd3c39b6c82123ec552ac825772